graphql/queries/org-list-outside-collaborators-by-repo.graphql

```diff
@@ -0,0 +1,25 @@
+query( $cursor: String) {
+    organization(login: "another-boring-org") {
```

Comment:
I'd prefer to follow the naming convention here with "ORG_NAME" so it's clear that it's a variable that is supposed to be replaced